litbamf/server.go

```diff
@@ -0,0 +1,21 @@
+package litbamf
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path"
+)
+
+func BamfListen(port uint16) {
+
+	listenString := fmt.Sprintf("127.0.0.1:%d", port)
+	ex, err := os.Executable()
```

Comment:
replace 13 to 18 with 
```
exPath := path.Dir(os.Args[0])
```
though this still makes the js relative to executable path which won't work for most uses